agent/api/task_test.go

```diff
@@ -214,6 +214,11 @@ func TestDockerHostConfigRawConfig(t *testing.T) {
 	assert.Nil(t, configErr)
 
 	expectedOutput := rawHostConfigInput
+	expectedOutput.CPUPercent = minimumCPUPercent
+	if minimumCPUPercent == 1 {
```

Comment:
You could check `runtime.GOOS == "windows"` if you want to make an OS assertion.